privacy/use-cases/src.md

```diff
@@ -0,0 +1,28 @@
+# EMV&reg; Secure Remote Commerce (SRC) User Recognition
+<sup>[Home][home] > [Use Cases][use-cases] > SRC User Recognition</sup>
+
+See more [information about EMV&reg;SRC](https://www.emvco.com/emv-technologies/src/), including the "Click-to-Pay" consumer facing UX.
+
+## Overview of Click-to-Pay Flow
+
+* The user pushes the Click-to-Pay button.
+* The SRC Initiator (SRC-I) determines (via a cookie, typically) whether this is a returning user, and if so finds the user's SRC identity (e.g., email address).
```

Comment:
This is interesting, as its not how I thought SRC worked! I had thought it was as follows:

1. Sometime in the past, user visits `https://src-a.com` and signs up for SRC with that system.
    * At this time, `https://src-a.com` drops a 1p cookie saying 'this user has ID abcd1234'
2. Later, user is on a site and clicks a Click To Pay button.
    1. SRC-I loads to handle the payment, and has no particular knowledge of the user
    2. SRC-I sends (e.g.) a fetch request to `https://src-a.com/get-details`
        * This fetch request is made with 3p cookies for `https://src-a.com`
        * Server-side, `https://src-a.com` reads its previously set cookie and looks up abcd1234 in its database to fetch credit card details
        * These details are then returned to SRC-I
    3. Repeat step ii. for each known SRC system.

Reply:
You may well be right. @tblachowicz would you be able to confirm? Thank you!

Reply:
Confirmed! @stephenmcgruer is right.

The SRCI does not have any particular knowledge of the consumer that allows to recognize the consumer. You could think about SRCI as you'd typically think about PSP which is _invisible_ to the consumer.

The SRC has a relationship with the Consumer and should be able to recognize the returning consumer. However, the SRCI orchestrates the recognition by calling `isRecognized()` on each SRC System.